src/app/shared/pipes/relativetime.ts

```diff
@@ -15,9 +15,9 @@
 import {Pipe, PipeTransform} from '@angular/core';
 import moment from 'moment';
 
-@Pipe({name: 'kmRelativeTime'})
+@Pipe({name: 'kmRelativeTime', pure: false})
```

Comment:
This is the solution. Small pipes can be marked as impure. That means it can produce different results with the same input what is exactly the case here.